src/lib/components/Placeholder.svelte

```diff
@@ -1,107 +1,107 @@
 <script context="module">
-	const DEFAULT_GRADIENTS = [
-		{
-			dur: `2s`,
-			values: '1; -2',
-			attributeName: 'offset',
-			repeatCount: 'indefinite',
-			stopColor: '#C0C0C0',
-			stopOpacity: '0.5',
-			offset: '50%',
-		},
-		{
-			dur: `2s`,
-			values: '2; -1',
-			attributeName: 'offset',
-			repeatCount: 'indefinite',
-			stopColor: '#EAEAEA',
-			stopOpacity: '0.5',
-			offset: '50%',
-		},
-		{
-			dur: `2s`,
-			values: '-1; 2',
-			attributeName: 'offset',
-			repeatCount: 'indefinite',
-			stopColor: '#C0C0C0',
-			stopOpacity: '0.5',
-			offset: '50%',
-		},
-	];
+  const DEFAULT_GRADIENTS = [
+    {
+      dur: `2s`,
+      values: '1; -2',
+      attributeName: 'offset',
+      repeatCount: 'indefinite',
+      stopColor: '#C0C0C0',
+      stopOpacity: '0.5',
+      offset: '50%',
+    },
+    {
+      dur: `2s`,
+      values: '2; -1',
+      attributeName: 'offset',
+      repeatCount: 'indefinite',
+      stopColor: '#EAEAEA',
+      stopOpacity: '0.5',
+      offset: '50%',
+    },
+    {
+      dur: `2s`,
+      values: '-1; 2',
+      attributeName: 'offset',
+      repeatCount: 'indefinite',
+      stopColor: '#C0C0C0',
+      stopOpacity: '0.5',
+      offset: '50%',
+    },
+  ];
 </script>
 
 <script>
-	import { uniqueId } from '../utils';
+  import { uniqueId } from '../utils';
 
-	/**
-	 * SVG Element `height` attribute value
-	 */
-	export let height = 200;
+  /**
+   * SVG Element `height` attribute value
+   */
+  export let height = '200';
 
-	/**
-	 * SVG Element `width` attribute value
-	 */
-	export let width = 200;
+  /**
+   * SVG Element `width` attribute value
+   */
+  export let width = '200';
```

Comment:
To keep backwards compat please use `number | string` as type for `height` and `width`.